pkg/addons/seaweedfs/install.go

```diff
@@ -114,14 +116,14 @@ func ensureService(ctx context.Context, kcli client.Client, serviceCIDR string)
 		}
 	}
 
-	if err := kcli.Create(ctx, obj); err != nil && !k8serrors.IsAlreadyExists(err) {
+	if err := kcli.Create(ctx, obj); err != nil {
```

Comment:
i think this will fail when seaweedfs is upgraded later and the service already exists.

Reply:
Its deleted above here https://github.com/replicatedhq/embedded-cluster/pull/2027/files#diff-1ba7f2b58909b59c50d80f035bb11b2a496751bf84ed2ea2cc675cfa84d11880R113